src/Transports.AspNetCore/AuthorizationParameters.cs

```diff
@@ -0,0 +1,68 @@
+#nullable enable
+
+using System.Security.Claims;
+using System.Security.Principal;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+
+namespace GraphQL.Server.Transports.AspNetCore;
+
+/// <summary>
+/// Authorization parameters
```

Comment:
It does not look clear. What is the appointment?

Reply:
This class is used by the `AuthorizationHelpers` class to perform arbitrary actions based on provided authentication properties/attributes/etc.  It is needed by the new authorization validation rule (will be a separate PR) so as to eliminate duplicate code.  It is not intended to be called from user code, but is `public` if the need arises.  I'm open to any suggestions as to the xml comments here.